src/main/scala/com/metl/model/Reputation.scala

```diff
@@ -25,9 +22,9 @@ object Reputation extends Logger{
   def addStanding(who:String,what:GainAction.Value):Unit = Stopwatch.time("Reputation:addStanding(%s,%s)".format(who,what),{
     val score = Informal.value(what)
     if (score > 0){
-      val currentStanding = standingMap(who)
+      val currentStanding = standingMap.getOrElseUpdate(who, Informal.standing(who))
```

Comment:
Uh, the original was just a get.  I'd change that to standingMap.get(who).getOrElse(0) to replicate the same behaviour as existing.